config/initializers/rack_attack.rb

```diff
@@ -21,6 +21,8 @@
       request.ip
     end
   end
+
+  Rack::Attack.blocklist_ip(ENV["RACK_ATTACK_BLOCKED_IPS"])
```

Comment:
have you tried with an array? should we parse `ENV["RACK_ATTACK_BLOCKED_IPS"]` before passing it as a parameter?